app/controllers/atmosphere/admin/virtual_machine_templates_controller.rb

```diff
@@ -17,13 +21,26 @@ def show
   def edit
   end
 
+  # GET /virtual_machine_templates/1/share
+  def share
+  end
+
+  # POST /virtual_machine_templates/1/export
+  def export
```

Comment:
I don't like methods which are not restfull here. Maybe you can use similar solution as @Nuanda used to create ApplianceType port mappings, configurations, etc.?